kodilla-collections-advanced/src/test/java/com/kodilla/collections/adv/exercises/homework/FlightFinderTestSuit.java

```diff
@@ -0,0 +1,154 @@
+package com.kodilla.collections.adv.exercises.homework;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class FlightFinderTestSuit {
+
+    @Test
+    public void findFlightsFrom() {
+        // given
+        FlightFinder flightFinder = new FlightFinder(FlightRepository.getFlightsTable());
+        List<Flight> resultFlightFrom = flightFinder.findFlightsFrom("Warszawa");
+
+        // when
+        List<Flight> expectedFlightsList = new ArrayList<>();
+        expectedFlightsList.add(new Flight("Warszawa", "Krakow"));
+        expectedFlightsList.add(new Flight("Warszawa", "Gdansk"));
+
+        // then
+        assertEquals(expectedFlightsList, resultFlightFrom);
+    }
+
+    @Test
+    public void testFindFlightFromWarszawa() {
+        // given
+        FlightFinder testFlightFinder = new FlightFinder(FlightRepository.getFlightsTable());
+
+        // when
+        List<Flight> foundFlightFrom = testFlightFinder.findFlightsFrom("Warszawa");
+
+        // then
+        for (Flight flight : foundFlightFrom) {
```

Comment:
w testch nie uzywamy zadnych petli, instrukcji warunkowych itp.
Musisz explicite to wyrazic. Zreszta po co ta metoda testowa? W poprzedniej testujesz to samo.